apps/docs/src/.vitepress/components/ComponentCard.vue

```diff
@@ -0,0 +1,61 @@
+<script lang="ts" setup>
+export type ComponentCardProps = {
+  /** Component name. */
+  name: string;
+  /** If true an "Implemented" badge will be shown, "Coming Soon" otherwise. */
+  implemented?: boolean;
+};
+
+const props = defineProps<ComponentCardProps>();
+</script>
+
+<template>
+  <div class="card">
+    <article class="card__content">
```

Comment:
when you merge the classes, you get the same effect -> flatter DOM :)

```suggestion
  <article class="card">
```